src/com/cdd/bao/template/ClipboardSchema.java

```diff
@@ -179,8 +185,33 @@ private static JSONObject formatAssignment(Schema.Assignment assn) throws JSONEx
 		}
 		json.put("values", jvalues);
 		
+		if (schvoc != null) 
```

Comment:
```
if (schvoc == null) return json;
```

Reply:
Having the nesting is a nuanced hint that the feature within is an optional use case within a list of actions... the fact that it's at the end is just a coincidence. If it were a mandatory item that was central to the functionality, I would've been more inclined to do it the way you suggested.